Pokedex/Pokedex/View Controllers/PokedexTableViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  PokedexTableViewController.swift
+//  Pokedex
+//
+//  Created by Moses Robinson on 1/25/19.
+//  Copyright © 2019 Moses Robinson. All rights reserved.
+//
+
+import UIKit
+
+class PokedexTableViewController: UITableViewController {
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableView.reloadData()
+    }
+
+    // MARK: - Table view data source
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return pokemonController.pokedex.count
+    }
+    
+    let reuseIdentifier = "PokemonCell"
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath)
+
+        let pokemon = pokemonController.pokedex[indexPath.row]
+        cell.textLabel?.text = pokemon.name.capitalized
+        
+//        pokemonController.fetchImage(for: pokemon, completion: { (data) in
+//            guard let data = data else { return }
+//            let image = UIImage(data: data)
+//            cell.imageView?.image = image
+//            tableView.reloadData()
```

Comment:
If you comment out only that `reload()` function then delete function is working